Framework/API/test/MatrixWorkspaceTest.h

```diff
@@ -1350,16 +1355,14 @@ class MatrixWorkspaceTest : public CxxTest::TestSuite {
 
 private:
   Mantid::API::MantidImage_sptr createImage(size_t width, size_t height) {
```

Comment:
@quantumsteve 
**Very minor** - Would it make sense to have `width` and `height` as `const`?
e.g.
`Mantid::API::MantidImage_sptr createImage(const size_t width, const size_t height) {...}`